components/ui/CompactFeature.vue

```diff
@@ -16,6 +17,8 @@ import { Component, Prop } from 'vue-property-decorator'
 @Component
 export default class extends Vue {
   @Prop({ type: String, default: '' }) icon
+  @Prop({ type: String, default: '' }) title
```

Comment:
Instead of defaulting in empty strings, let's make the properties completely optional as it's the case with the icon, and check for them to prevent the title and the paragraph from showing if there is no content. 
```suggestion
  @Prop({ type: String }) title: string|undefined
```

I'm including the type for another refactor I'm proposing later.